tests/phpunit/tests/feed/atom.php

```diff
@@ -51,6 +53,17 @@ public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
 			wp_set_object_terms( $post, self::$category->slug, 'category' );
 		}
 
+		self::$post_id = $factory->post->create();
+
+		// Create a comment
```

Comment:
```suggestion
		// Create a comment.
```
Period at the end of the comment for consistency with the rest of the codebase.